### ISSUE TITLE
Printing

### ISSUE THREAD

**Issue by yaras6** [CoderLine/alphaTab#124]
On player demo first page is properly printed. Rest pages are blank. How to make printing work properly?

**Comment by Danielku15** [comment]
Looks like a problem of multiple features: 
- The responsive width is not suitable for printing. alphaTab will render into the available width, I tried some tricks to listen for the print event and resize alphaTab in this case but it seems browsers do not wait until the resize of alphaTab is finished. 
- alphaTab uses a special caching/virtualizing mechanism. At the time when alphaTab was rendered, not all parts of the music sheet are actually shown. Only when scrolling them into the viewport they are dynamically added. Here a screenshot from the DOM: 
![image](https://cloud.githubusercontent.com/assets/674916/23918848/dba879f8-08f4-11e7-8d09-079e014ff09d.png)

The solution is to render alphaTab into a popup window matching A4 screen size and then trigger the printing. I added a new feature that covers this workflow. Notice the new print button on the player sample: http://demo.alphatab.net/player.html
